packages/builder/src/components/automation/AutomationBuilder/FlowChart/TestDataModal.svelte

```diff
@@ -152,7 +150,7 @@
   {#if selectedValues}
     <div class="tab-content-padding">
       <AutomationBlockSetup
-        testData={$memoTestData}
+        bind:testData
```

Comment:
We're trying to steer away from using `bind` when working with props in the builder. It can lead to issues with mutation.

Reply:
I assume that, but it would require quite a lot of reactors around the stores... Given that we are going to type/cleanup the stores soon, happy if we check it then?